api/src/main/java/org/openmrs/module/sync/api/db/hibernate/HibernateSyncInterceptor.java

```diff
@@ -1155,14 +1157,12 @@ private void processPersistentCollection(AbstractPersistentCollection collection
 		Package pkg = new Package();
 		entriesHolder = new LinkedHashMap<String, OpenmrsObject>();
 		try {
-
-			CollectionMetadata collMD = getCollectionMetadata(owner.getClass(), ownerPropertyName, getSessionFactory());
-			if (collMD == null) {
-				throw new SyncException("Can't find a collection with " + ownerPropertyName + " in class "
-				        + owner.getClass());
+			Class elementClass = getCollectionAttributeType(owner.getClass(), ownerPropertyName, getSessionFactory());
+			if (elementClass == null) {
+				throw new SyncException(
+						"Can't find a collection with " + ownerPropertyName + " in class " + owner.getClass()
+				);
 			}

```

Comment:
This and the below is needed due to another Hibernate API change.  It seems to work in all of the unit tests.